CHANGELOG.md

```diff
@@ -24,7 +28,7 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [dacaiguoguogmail](https://github.com/dacaiguoguogmail)
   [#8461](https://github.com/CocoaPods/CocoaPods/issues/8461)
 
-* Set the path of development pod groups to root directory of the Pod 
+* Set the path of development pod groups to root directory of the Pod  
```

Comment:
change intended?

Reply:
Yep, it fixes that line (should also be two spaces)

Reply:
ah yea